### PR TITLE
[CPDLP-2397] Remove CalculationOrchestrator

### PR DESCRIPTION
**Pull request by leandroalemao**
### Context

- Ticket: [CPDLP-2397](https://dfedigital.atlassian.net/browse/CPDLP-2397)

### Changes proposed in this pull request

Remove `CalculationOrchestrator` as it isn't used anywhere.

[CPDLP-2397]: https://dfedigital.atlassian.net/browse/CPDLP-2397?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: spec/features/finance/payment_breakdown_spec.rb [hunk 1]
```diff
@@ -205,69 +177,57 @@ def then_i_should_see_correct_breakdown_summary
...
 
       expect(page).to have_content("VAT")
-      expect(page).to have_content(number_to_pounds(total_vat_breakdown))
+      expect(page).to have_content(number_to_pounds(total_vat))
     end
   end
 
   def then_i_should_see_the_correct_output_fees
     expect(page).to have_content("Output payments")
-    expect(page).to have_content(number_to_pounds(jan_starts_breakdowns[:output_payments][0][:per_participant]))
```
Comment (ethax-ross) [inline spec/features/finance/payment_breakdown_spec.rb:223]: Are these specs all just duplicating other functionality/the specs below them? I don't quite get how they were passing before given the services weren't referenced anywhere 😕

**Review by ethax-ross — APPROVED** [review]
Looks good; minor question on the specs but the changes look fine.

File: spec/features/finance/payment_breakdown_spec.rb [hunk 2]
```diff
@@ -205,69 +177,57 @@ def then_i_should_see_correct_breakdown_summary
...
-    expect(page).to have_content(jan_starts_breakdowns[:output_payments][0][:participants])
-    expect(page).to have_content(number_to_pounds(jan_starts_breakdowns[:output_payments][0][:subtotal]))
+    expect(page).to have_content(number_to_pounds(jan_statement_calculator.additions_for_started))
+    expect(page).to have_content(jan_statement_calculator.started_count)
+    expect(page).to have_content(number_to_pounds(jan_statement_calculator.output_fee))
+
+    expect(page).to have_content(number_to_pounds(nov_statement_calculator.additions_for_started))
```
Comment (cwrw) [inline spec/features/finance/payment_breakdown_spec.rb:199]: I think we might only need to have January statement checks here, as November statement is selected later in the steps?

File: spec/features/finance/payment_breakdown_spec.rb [hunk 3]
```diff
@@ -205,69 +177,57 @@ def then_i_should_see_correct_breakdown_summary
...
-    expect(page).to have_content(jan_starts_breakdowns[:other_fees][:uplift][:participants])
-    expect(page).to have_content(number_to_pounds(jan_starts_breakdowns[:other_fees][:uplift][:subtotal]))
-  end
+    expect(page).to have_content(number_to_pounds(jan_statement_calculator.uplift_additions_count))
+    expect(page).to have_content(number_to_pounds(jan_statement_calculator.uplift_fee_per_declaration))
+    expect(page).to have_content(number_to_pounds(jan_statement_calculator.uplift_additions_count * jan_statement_calculator.uplift_fee_per_declaration))
 
-  def number_of_declarations
-    jan_starts_breakdowns[:output_payments].map { |params| params[:participants] }.inject(&:+) + jan_retained_breakdowns[:output_payments].map { |params| params[:participants] }.inject(&:+)
+    expect(page).to have_content(number_to_pounds(nov_statement_calculator.uplift_additions_count))
```
Comment (cwrw) [inline spec/features/finance/payment_breakdown_spec.rb:212]: same question here around requiring only January?

**Pull request merged**